src/axom/primal/operators/intersect.hpp

```diff
@@ -323,6 +328,30 @@ AXOM_HOST_DEVICE bool intersect(const Ray<T, DIM>& R,
   return detail::intersect_ray(R, bb, ip);
 }
 
+/*!
+ * \brief Computes the intersection of the given line, L, with the Box, bb.
+ *
+ * \param [in] L the specified line (two-sided ray)
+ * \param [in] bb the user-supplied axis-aligned bounding box
+ *
+ * \param [out] ip the intersection point where L intersects bb.
```

Comment:
This method follows the analogous ray-bb intersection routine, which returns the intersection with the minimum `t` value as the returned point. While doing this in the line case isn't quite as conceptually straightforward, since that parameter will be negative when the origin is interior to the bounding box, it's consistent, and does serve as some kind of guarantee. 

At the same time, my own usage of this method doesn't even use the point of intersection at all, so it's sort of a waste to compute it. With this in mind, I'll update the comment to reflect the behavior of this method, but also write a new one that doesn't compute the intersection in physical space.